app/models/merchant.rb

```diff
@@ -5,21 +5,23 @@ class Merchant < ApplicationRecord
   has_many :customers, through: :invoices
   has_many :transactions, through: :invoices
 
+  enum status: { disabled: 0, enabled: 1 }
+
   def top_five_customers
     customer_ids = invoices.joins(:transactions)
-                           .where("transactions.result = 1")
+                           .where("transactions.result = ?", Transaction.results[:success])
```

Comment:
I am really happy we took the time to fix the enums. The helper methods are so much better than hardcoding the numbers. Thanks for making this a point!